core/trashcan.js

```diff
@@ -166,8 +166,9 @@ Blockly.Trashcan.prototype.createDom = function() {
   */
   this.svgGroup_ = Blockly.utils.createSvgElement('g',
       {'class': 'blocklyTrash'}, null);
+  var clip;
```

Comment:
While it's really tempting to do this (I have the same impulse), we try to not introduce unnecessary changes to this repository to make it easier to merge with Blockly. Please return the repeated var declaration.